charts/oes/charts/spinnaker/templates/rbac/spinnaker-sa.yaml

```diff
@@ -14,8 +14,8 @@ metadata:
 {{ include "spinnaker.standard-labels" . | indent 4 }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: cluster-admin
+  kind: Role             # ClusteRoleBinding if we have access accross the cluster
```

Comment:
If we change this to namespace scoped, Spinnaker cannot deploy any resource to any other namespace in the cluster

Reply:
As discussed, if user wants Spinnaker to deploy to other namespaces in the cluster. A kubeconfig file should be used to add another account.